src/org/zaproxy/zap/extension/ascanrulesAlpha/FormatString.java

```diff
@@ -98,25 +90,21 @@ public int getCategory() {
 
 	@Override
 	public String getSolution() {
-		if (vuln != null) {
-			return vuln.getSolution();
-		}
-		return "Failed to load vulnerability solution from file";
+		return Constant.messages.getString(MESSAGE_PREFIX + "soln");
 	}
 
 	@Override
 	public String getReference() {
-		if (vuln != null) {
-			StringBuilder sb = new StringBuilder();
-			for (String ref : vuln.getReferences()) {
-				if (sb.length() > 0) {
-					sb.append("\n");
-				}
-				sb.append(ref);
-			}
-			return sb.toString();
-		}
-		return "Failed to load vulnerability reference from file";
+		return Constant.messages.getString(MESSAGE_PREFIX + "refs");
+	}
+	
+	@Override
+	public boolean targets(TechSet technologies) { 

```

Comment:
It would be great if the method were kept in the same place (easier to review and better for the history).